mlir/test/Target/LLVMIR/Import/intrinsic.ll

```diff
@@ -865,6 +865,14 @@ define float @ssa_copy(float %0) {
   ret float %2
 }
 
+; CHECK-LABEL: llvm.func @nvvm
```

Comment:
Can you move the tests into an nvvmir.ll file in the same folder to follow the same structure as in the export?

Can you also add a test for every intrinsic? I know it seems like a lot of copy paste but it still seems helpful to exercise the import of all of them in case things get refactored in LLVM land for example.